ci.cue

```diff
@@ -87,6 +89,43 @@ dagger.#Plan & {
 			command: flags: "-race": true
 		}
 
+		integration: bats.#Bats & {
+			_daggerLinuxBin: build.go & {
```

Comment:
This is used a regular binary rather than the debug binary

Reply:
Do you mean it **must** be the debug binary here? I would assume so?

Reply:
Yes, `dagger-debug` in the Makefile